conf/detekt.yml

```diff
@@ -262,8 +262,9 @@ formatting:
         active: false
     AnnotationSpacing:
         active: true
+    # todo fix and enable
     ArgumentListWrapping:
```

Comment:
https://github.com/pinterest/ktlint/pull/1196
This is fixed too

Reply:
That's why I didn't update the Detekt version [here](https://github.com/avito-tech/avito-android/pull/1163) :)

Reply:
We need to run ktlint separately i think
MBS-11763